test/e2e/e2e-service-provider/src/main/java/test/apache/skywalking/e2e/User.java

```diff
@@ -16,7 +16,7 @@
  *
  */
 
-package org.apache.skywalking.e2e;
+package test.apache.skywalking.e2e;
```

Comment:
What does the renaming fix? If it really fixes something, please add the directory to

https://github.com/apache/skywalking/blob/8e6a527f78108c7a466ba2dd87eaf5c47d1765cc/tools/coverage/report.sh#L32-L34

i.e. `sudo rm -rf "${JACOCO_HOME}"/classes/"${exec_data}"/test || true `

Reply:
So the package name rules need to be updated too, is the renaming a must?

Reply:
Use the `org.apache.skywalking` package prefix, then the `SpringMVC` plugin cannot be enhanced, because that package is ignored. So avoid using this package.

Reply:
@kezhenxu94  I seem the e2e also has the style check, and It's must match the package name with regex, `^org\.apache\.skywalking(\.[a-zA-Z][a-zA-Z0-9]*)+$`, so how can I fix that?

Reply:
> Use the `org.apache.skywalking` package prefix, then the `SpringMVC` plugin cannot be enhanced, because that package is ignored. So avoid using this package.

Then you need to:

- update the package name rules:

https://github.com/apache/skywalking/blob/dcd66ee6e940bdc9a7ecf90415929bfd48a3a284/apm-checkstyle/checkStyle.xml#L99

- add the package to exclusion list

https://github.com/apache/skywalking/blob/8e6a527f78108c7a466ba2dd87eaf5c47d1765cc/tools/coverage/report.sh#L32-L34

i.e. `sudo rm -rf "${JACOCO_HOME}"/classes/"${exec_data}"/test || true `

Reply:
I think test.apache.skywalking package is used in the spring plugin test too.

Reply:
> I think test.apache.skywalking package is used in the spring plugin test too.

Yes.

Reply:
> > Use the `org.apache.skywalking` package prefix, then the `SpringMVC` plugin cannot be enhanced, because that package is ignored. So avoid using this package.
> 
> Then you need to:
> 
> * update the package name rules:
> 
> https://github.com/apache/skywalking/blob/dcd66ee6e940bdc9a7ecf90415929bfd48a3a284/apm-checkstyle/checkStyle.xml#L99
> 
> * add the package to exclusion list
> 
> https://github.com/apache/skywalking/blob/8e6a527f78108c7a466ba2dd87eaf5c47d1765cc/tools/coverage/report.sh#L32-L34
> 
> i.e. `sudo rm -rf "${JACOCO_HOME}"/classes/"${exec_data}"/test || true `

I have changed, please take a look.